apps/android/app/src/main/res/values-de/strings.xml

```diff
@@ -358,15 +358,15 @@
   <string name="how_to_use_simplex_chat">Wie man SimpleX nutzt</string>
   <string name="markdown_help">Markdown Hilfe</string>
   <string name="markdown_in_messages">Markdown in Nachrichten</string>
-  <string name="chat_with_the_founder">Senden Sie Fragen und Ideen</string>
+  <string name="chat_with_the_founder">Senden Sie Ihre Fragen und Ideen</string>
   <string name="send_us_an_email">Senden Sie uns eine E-Mail</string>
   <string name="chat_lock">SimpleX Sperre</string>
   <string name="chat_console">Chat Konsole</string>
   <string name="smp_servers">SMP-Server</string>
   <string name="install_simplex_chat_for_terminal">Installieren Sie <xliff:g id="appNameFull">SimpleX Chat</xliff:g> als Terminalanwendung</string>
-  <string name="star_on_github">Stern auf GitHub</string>
-  <string name="contribute">Beitragen</string>
-  <string name="rate_the_app">Bewerte die App</string>
+  <string name="star_on_github">Einen Stern auf GitHub vergeben</string>
```

Comment:
Same here, as well as on iOS. For clarity - most of the text keys in this PR are list items in the settings menu - `contribute`, `rate_the_app` too